ceph-build/build/build_rpm

```diff
@@ -34,7 +34,7 @@ cp -a rpm/*.patch ${BUILDAREA}/SOURCES/. || true
 # Build RPMs
 BUILDAREA=`readlink -fn ${BUILDAREA}`   ### rpm wants absolute path
 cd ${BUILDAREA}/SPECS
-rpmbuild -ba --define "_topdir ${BUILDAREA}" ceph.spec
+rpmbuild -ba --define "_topdir ${BUILDAREA}" ${CEPH_EXTRA_RPMBUILD_ARGS} ceph.spec
```

Comment:
Yeah, those were earlier builds; I've repushed, removing the '-' for rpmbuild, but didn't hand-update ceph-dev-new-*.  I'll do that.